packages/bitcoin-node-wallet-provider/lib/BitcoinNodeWalletProvider.js

```diff
@@ -0,0 +1,122 @@
+import WalletProvider from '@liquality/wallet-provider'
+import JsonRpcProvider from '@liquality/jsonrpc-provider'
+import BitcoinNetworks from '@liquality/bitcoin-networks'
+import * as bitcoin from 'bitcoinjs-lib'
+import { sha256 } from '@liquality/crypto'
+import { Address, addressToString } from '@liquality/utils'
+
+import { version } from '../package.json'
+
+const BIP70_CHAIN_TO_NETWORK = {
+  'main': BitcoinNetworks.bitcoin,
+  'test': BitcoinNetworks.bitcoin_testnet,
+  'regtest': BitcoinNetworks.bitcoin_regtest
+}
+
+const ADDRESS_TYPES = ['legacy', 'p2sh-segwit', 'bech32']
```

Comment:
Can we export this somewhere and reuse it? `BitcoinAddressTypes`?

Reply:
Moved to `bitcoin-utils`